plans/main.fmf

```diff
@@ -22,7 +22,10 @@ adjust+:
         SYSTEM_RELEASE: oracle-8.4
       when: distro == oraclelinux-8.4
     - environment+:
-        SYSTEM_RELEASE: oracle-8.6
+        #TODO bump distro context to 8.7 when the image is available on the testing farm
+        # Even though the distro context is 8.6 we are setting the SYSTEM_RELEASE to 8.7
+        # to tag the updated system correctly
```

Comment:
Maybe something similar to this comment could live in `.packit.yaml` too. I imagine someone (me) looking over the packit.yaml and not getting why we set ol-8.6 and we test for ol-8.7

Reply:
You are right, I wonder why I did not comment in the `.packit.yaml` in the first place :thinking: